docs/getting-started/github-apps.rst

```diff
@@ -51,15 +46,51 @@ How the Dockstore GitHub App works
 ----------------------------------
 
 With the Dockstore GitHub App, the synchronization is done automatically. When
-you publish a new release of a service on GitHub, Dockstore is notified,
-and Dockstore updates its copy of the service. After you publish a new release
-of a service on GitHub, a new version of the service will be present in
+you publish a new release of a workflow on GitHub, Dockstore is notified,
+and Dockstore updates its copy of the workflow. After you publish a new release
```

Comment:
`publish a new release...` -- now that you've added branch and delete support, might want to word this more generically. Or say "for example" earlier on if you're not trying to cover all cases here.